libs/packages/components/src/lib/collapse/collapse.directive.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { CollapseDirective } from './collapse.directive';
+
+describe('CollapseDirective', () => {
+  it('should create an instance', () => {
+    const directive = new CollapseDirective();
+    expect(directive).toBeTruthy();
+  });
+});
```

Comment:
Tests need to be more comprehensive to test that the class is being applied and removed based on the variable being toggled.